pkg/cluster/cluster_config.go

```diff
@@ -120,3 +121,31 @@ func GetPlatform(cli client.Client) (Platform, error) {
 	// check and return whether ODH or self-managed platform
 	return isSelfManaged(cli)
 }
+
+type Release struct {
+	Name    Platform                `json:"name,omitempty"`
+	Version version.OperatorVersion `json:"version,omitempty"`
+}
+
+func SetRelease(cli client.Client) (*Release, error) {
```

Comment:
I personally do not see reason for pointer (*Release) here since you anyway dereference it right after the call and optimiser should avoid extra structure copying, but it's a matter of taste, probably.